bucketeer/src/main/kotlin/io/bucketeer/sdk/android/BKTEvaluation.kt

```diff
@@ -1,26 +1,13 @@
 package io.bucketeer.sdk.android
 
+import io.bucketeer.sdk.android.internal.model.ReasonType
+
 data class BKTEvaluation(
   var id: String,
   val featureId: String,
   val featureVersion: Int,
   val userId: String,
   val variationId: String,
   val variationValue: String,
-  val reason: Reason,
-) {
-  enum class Reason(val value: Int) {
```

Comment:
`io.bucketeer.sdk.android.internal.model.ReasonType` is placed in `internal` package and we don't want to expose it to user-facing API.

So I'd rather want to keep it separate.

Reply:
@yshrsmz  the current implement duplicates `ReasonType`  as `Reason`, I see no benefit when repeating it.
we may move ReasonType to public. 
what do you think?

Reply:
or maybe we only keep the raw value as "String" as `Reason` to not expose the internal class

Reply:
> the current implement duplicates ReasonType as Reason, I see no benefit when repeating it.
> we may move ReasonType to public.

Sounds good to me.　👍

Reply:
@yshrsmz, to keep consistency, I think we should use the `BKT` as a prefix for the public classes. Not sure if this case is okay not having the prefix. What do you think?
I was thinking `val reason: BKTEvaluationReason`.

Reply:
On second thought, we might want to keep them separate as before.

It was duplicated because I wanted to separate user-facing API from internal/remote API representations so that they could act as an anti-corrunption layer, or could have different implementations.

Reply:
To be honest, I prefer to separate it as you did before to avoid future breaking changes with API changes.

Reply:
I checked the iOS reference again, and `ReasonType` is public on iOS.
So to keep it simple and consistent, I think it will be  okay to go with `BKTEvaluationReason`  
@cre8ivejp  
cc: @yshrsmz